9000/alpine/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM java:8-jre-alpine
+
+RUN apk --update upgrade && \
```

Comment:
`apk --no-cache` I think would be better here, then should be able to forgo the rm later.